js/blocks/components/fetch-input.js

```diff
@@ -379,7 +406,6 @@ class FetchInput extends Component {
 						</div>
 					</Popover>
 				}
-				{ ! showSuggestions || '' === inputValue || ! results.length && this.setInputValidity( false ) }
```

Comment:
These `setInputValidity()` calls are moved to `componentDidUpdate()`, though the logic that calls them is changed.